pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_SalMultiLanguageTests"
+description = "SAL tests to verify that publishers and subscribers can communicate between all the programming languages, C++, Java and SalObj."
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = { documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_SalMultiLanguageTests" }
+dynamic = [ "version" ]
+dependencies = []
+
+[tools.setuptools]
+package-data = {"" = "*.csv"}
```

Comment:
This package doesn't contain any csv files so please remove this section.

Reply:
Removed.